src/wordscan/WordScan.java

```diff
@@ -50,7 +50,7 @@ public static void main(String[] args) {
 				}
 				if (count != 0) // Check for count not equal to zero
 				{
-					System.out.println("The pattern is present " + count + " times in the file");
+					System.out.println("The word is present " + count + " times in this file");

```

Comment:
Changing from "pattern" to "word" maybe is strange since it is patterns that it should look for. However, this is just a small thing.

Reply:
Its resolved